File: BLPS_Lab1/src/main/java/org/example/blps_lab1/dto/HotelDto.java
package org.example.blps_lab1.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.extern.slf4j.Slf4j;

import java.util.Date;

@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
@Slf4j
public class HotelDto {
    private String hotelName;
    private Float price;
    private Date dayOfArrival;
    private Date dayOfDeparture;
    private Integer countOfGuests;
    private String address;
}

File: BLPS_Lab/src/main/kotlin/org/example/blps_lab/dto/HotelDto.kt
package org.example.blps_lab.dto

import java.math.BigDecimal
import java.util.*

data class HotelDto (
    val hotelName: String,
    val price: BigDecimal,
    val dayOfArrival: Date,
    val dayOfDeparture: Date,
    val guestCount: Int,
    val address: String
)